app/controllers/products_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ProductsController < ApplicationController
+  def index
+    @products = Stripe::Product.list
+  end
```

Comment:
@husseinagrama ...Let's write an `RSPEC` test for this controller action next Thursday. I'll add it to the Trello card.